types/fluxible-addons-react/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for fluxible-addons-react 0.2
```

Comment:
There should be a file `types/fluxible-addons-react/fluxible-addons-react-tests.ts` with an example usage of this package.

Reply:
thx for notificing.
I've added the test file, pleaste take a look

Reply:
Thanks!